extensions/apps/profile/src/components/pages/edit-profile/index.tsx

```diff
@@ -207,6 +207,12 @@ const EditProfilePage: React.FC<EditProfilePageProps> = props => {
     <Stack direction="column" spacing="gap-y-4" customStyle="h-full">
       <Card radius={20} customStyle="py-4 h-full">
         <EditProfile
+          cancelButtonLabel={t('Cancel')}
+          leavePageLabel={t('Leave page')}
+          modalTitle={t('Unsaved changes')}
+          modalDescription={t(
+            "Are you sure you want to leave this page? The changes you've made will not be saved.",
+          )}
```

Comment:
I don't think its necessary to pass all this props. You can use `useTranslation` inside the `EditProfile` component.